orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -144,6 +144,20 @@ struct OrtTrainingApi {
                   size_t inputs_len, _In_reads_(inputs_len) const OrtValue* const* inputs,
                   size_t outputs_len, _Inout_updates_all_(outputs_len) OrtValue** outputs);
 
+  /** \brief Set the learning rate for this training session.
+  *
+  * This function allows users to set the learning rate for the training session. The training
+  * session by default starts with a learning rate of 0.001. It can be overwritten by invoking
```

Comment:
nit: listing the 0.001 value could be easily lost of track once the default learning rate get changed (in another file).

Reply:
How else shall we document the default learning rate? It might not be convenient for users to read the `Optimizer` cpp implementation to find the default learning rate.